Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -109,7 +109,9 @@ public void SetContextDocument(object doc)
       }
     }
 
-    public void SetContextObjects(List<ApplicationObject> objects) => ContextObjects = objects.ToDictionary(x => x.applicationId);
+    //NOTE: not all objects come from Revit, so their applicationId might be null, in this case we fall back on the Id
+    //this fallback is only needed for a couple of ToNative conversions such as Floor, Ceiling, and Roof 
+    public void SetContextObjects(List<ApplicationObject> objects) => ContextObjects = objects.ToDictionary(x => x.applicationId ?? x.OriginalId);
     public void SetPreviousContextObjects(List<ApplicationObject> objects) => PreviousContextObjects = objects.ToDictionary(x => x.applicationId);
```

Comment:
We will also need to update the `SetPreviousContextObjects` in the same way

Reply:
true!